### PR TITLE
Update dependencies including webview_flutter to 3.0.0

### PR DESCRIPTION
**Pull request by NatsuOnFire**

### CODE REVIEW

**Comment by NatsuOnFire** [comment]
@dotronglong Can you validate my PR or tell me if anything is missing?

File: example/lib/main.dart [hunk 1]
```diff
@@ -34,7 +34,7 @@ class _MyHomePageState extends State<MyHomePage> {
       clientSecret: '709fb6441354c8d148248ae2cab0673b4ce7f1d5',
       redirectUrl: 'https://l2t-flutter.firebaseapp.com/__/auth/handler',
       title: 'GitHub Connection',
-      centerTitle: false
+      centerTitle: false,
```
Comment (dotronglong) [inline example/lib/main.dart:37]: hi, why do we need this extra `,`?

**Comment by dotronglong** [comment]
hi @NatsuOnFire I see 2 major things and I would like to ask you for what reason you think we should have this PR?
- Change user agent, does it mean the old agent won't work any more?
- Change webview_flutter to higher version 2.x to 3.x, is it necessary and is there any compatible problems? Did you test on all major devices on both iOS and android?

Until now, please correct me if I am wrong I don't see a big reason to approve this PR.

File: example/lib/main.dart (hunk 1 above)
Comment (NatsuOnFire) [reply example/lib/main.dart:37]: It's to keep the indentation

**Comment by NatsuOnFire** [comment]
User-agent : 
It's just to have the latest version, good practice to be up to date. I can pull it off if you want

Update webview_flutter 3.x : 
In my app I am also using webview_flutter and I cannot use the latest version because this package uses a lower version.
Only 1 change in 3.0, on Android, hybrid composition (SurfaceAndroidWebView) is now the default.
Tested on Android and IOS, it works fine

**Comment by dotronglong** [comment]
@NatsuOnFire thank you for your information. Since I don't have time to test on all of major devices, it's hard for me to approve, where other people might face a problem with this PR. However, I read document regarding webview_flutter and it makes sense for me to approve the PR. In addition, can you please revert user agent, as I don't think it is necessary.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,8 @@
+## [0.0.5-dev.4] - v0.0.5-dev.4
+
+* Update default user-agent
```
Comment (dotronglong) [inline CHANGELOG.md:3]: Once you update by removing the user-agent changes, please help to update the changelog to express you update webview_flutter to version 3.x as well. Thank you @NatsuOnFire

**Comment by NatsuOnFire** [comment]
@dotronglong I updated the PR 😉

File: CHANGELOG.md (hunk 2 above)
Comment (dotronglong) [reply CHANGELOG.md]: @NatsuOnFire can you help to update based on above suggestion?

**Review by dotronglong — APPROVED** [review]

**Pull request merged**